tmt-app/src/Data/BobaData.js

```diff
@@ -0,0 +1,438 @@
+export const CareersData = [
```

Comment:
I suggest that components should be capitalized, constants be uppercase, everything else be camel-case.

i.e. export const CAREERS_LIST = [...]

Reply:
Title of the file included

Reply:
+1 to this. JS doesn't have true immutable constants, so creating a shared method of communication for a predefined constant with cap snake case is a great way to have that agreement and understanding of what this is expected to be.

another common one is PascalCasing for components. and camelCasing for all other variables and functions.